.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,92 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["Triage: Needed"]
+body:
+  - type: markdown
+    attributes:
+      value:
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: happened
+    attributes:
+      label: What happened?
+      description: "Tell us what happened. Provide a detailed description of the issue with context."
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: What you expected?
+      description: "Tell us what you expected to happen."
+    validations:
+      required: true
+  - type: textarea
+    id: possible-fix
+    attributes:
+      label: Possible fix?
+      description: "Tell us about changes that resolve or influence the issue."
+  - type: textarea
+    id: repro
+    attributes:
+      label: Steps to reproduce?
+      description: "Tell us how to reproduce the issue. Walk us through it step by step."
+      value:
+        1.
+        2.
+        3.
+        ...
```

```suggestion
      placeholder: |
        1.
        2.
        3.
        ...
```

this fit better as placeholder instead of an hardcoded value,
when the field value is not inline with the field itself must be start with a pipe `|`